videos/utils.py

```diff
@@ -110,3 +111,52 @@ def update_metadata(source_obj, new_uid, new_s3_path):
     new_metadata["uid"] = new_uid
     new_metadata["file"] = str(new_s3_path).lstrip("/")
     return new_metadata
+
+
+def get_course_tag(website: Website) -> str:
+    """
+    Get the course URL slug from the website's url_path.
+
+    Args:
+        website: Website instance
+
+    Returns:
+        str: The course URL slug (e.g., '18-01-fall-2020')
+    """
+    url_path = website.url_path
+    if url_path.startswith("courses/"):
```

Comment:
It's not ideal to hardcode `courses` here; this is actually the value of the `WEBSITE_CONFIG_ROOT_URL_PATH_KEY` variable in the starter config. There is a function to retrieve this: https://github.com/mitodl/ocw-studio/blob/805c9c67ca7a060856ddf0ba66cae53e68230503/websites/site_config_api.py#L98